app/styles/components/category-item.scss

```diff
@@ -13,7 +13,7 @@
     font-size: $body-font-size-small;
     margin: 10px 0;
 
-    &.selected {
+    .selected & {
       color: $default-color;
```

Comment:
@JoshSmith With the change I made, this css would apply to the description. Once you added your change, though, this is no longer the case, so this style block does nothing, as far as I can tell. 

This means that the `description` text is always using the `$text--light` (`#666`) color and does not switch to `$default-color` (`#08A8FC` when selected.

I'm not against your approach and we probably want BEM to apply. However, if we want the description color to change, then we want the class to bind to the component, not the component icon. Otherwise, we need to bind to the description as well, so we're back to square one.

If you're ok with this behavior as is (where the description stays `#666`, regardless of status, then we should at the very least remove this style block, since it does nothing.